### DISCUSSION TITLE
Why my port is not installing anything inside of the vcpkg installed folder?

### DISCUSSION THREAD

**Discussion by al-sabr** [microsoft/vcpkg#22050] [Q&A] [answered]
I am porting a simple header + .lib file library to CMake and vcpkg

### Links for the related projects

Port : https://github.com/al-sabr/flash-runtime-extensions
CMake : https://github.com/al-sabr/FlashRuntimeExtensions

When I am trying locally to install the port via:

`vcpkg install flash-runtime-extensions:x64-windows`

The problem I get is why doesn't **vcpkg** create and copy the related folder & files inside of :

`g:\vcpkg\installed\x64-windows\include\flash-runtime-extensions\FlashRuntimeExtensions.h`
`g:\vcpkg\installed\x64-windows\debug\lib\FlashRuntimeExtensions.lib`

This is what I end up with

![image](https://user-images.githubusercontent.com/26530975/146384244-f3789f17-6ffd-4340-91c3-378b6b54d813.png)

**Comment by autoantwort** [comment]
But the vcpkg install command is successful?

**Reply by al-sabr** [reply]
Yes it is successful

**Comment by al-sabr** [comment, marked as answer]
This can be closed I resolved the situation.
